packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -131,6 +131,18 @@ cluster.addAutoScalingGroup(autoScalingGroup);
 
 If you omit the property `vpc`, the construct will create a new VPC with two AZs.
 
+By default, all machine images will auto-update to the latest version
+on each deployment, causing a replacement of the instances in your AutoScalingGroup.
```

Comment:
```suggestion
on each deployment, causing a replacement of the instances in your AutoScalingGroup if the AMI has been updated since the last deployment.
```